index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1 id="hello">Hello World, we are software developers!</h1>
+    <h1 id="title">Survey Form</h1>
+    <p id="description"><b>This a very important survey. Please participate.</b></p>
+    <form action="" id="survey-form">
+     
+      <label for="name" id="name-label"><b>Name: </b></label>
+      <input type="text" id="name" placeholder="Enter your name"> <br> <br>
+
+      <label for="emial" id="email-label"><b>Email: </b></label>
```

Comment:
- [ ] You've done a really great job in your feedback so far. However, it would be even more helpful if you could phrase your comments politely as suggestions rather than orders. Using a friendly and constructive tone will encourage the recipient to consider your feedback positively and make it easier for them to act on the suggestions. Thank you for your attention to this important detail! 😊

Reply:
Thanks for the suggestion, i will change them for sure

Reply:
- [ ] Great work, comment is polite and clear 👏🏾